### PR TITLE
Feat: refactored some code

### PR DESCRIPTION
**Pull request by riimuru**
- fixed slug search on the `getAnime`  function bc some animes were not being found e.g: code geass season 2.
- some animes contain multiple quality links and the best quality link is always the last one on the list.
- refactored recommended anime card component on the watch page.
    -  <details><summary>before</summary></details>
    - <details><summary>after</summary></details>

### CODE REVIEW

**Comment by chirag-droid** [comment]
I think the compiler used by NextJs by default is faster than babel. https://nextjs.org/docs/advanced-features/compiler

File: utility/gogoanime.js [hunk 1]
```diff
@@ -1,26 +1,17 @@
-import { scrapeMP4 } from "gogoanime-api/lib/anime_parser"
-const cheerio = require("cheerio")
+import { scrapeMP4, scrapeSearch } from "gogoanime-api/lib/anime_parser";
+const cheerio = require("cheerio");
```
Comment (chirag-droid) [inline utility/gogoanime.js:2]: cheerio is an unnecessary import

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (chirag-droid) [inline .babelrc:1]: Remove babelrc, as nextjs default compiler is faster

**Comment by riimuru** [comment]
done 👍

**Comment by chirag-droid** [comment]
Thanks! I am merging this with refactor branch

**Comment by riimuru** [comment]
I recommend merging it witth main since it will fix animes not loading.

**Comment by chirag-droid** [comment]
Now I have merged it with main branch. I wanted to fix a visual bug in recommended animes and also show anime description.

**Comment by riimuru** [comment]
> Now I have merged it with main branch. I wanted to fix a visual bug in recommended animes and also show anime description.
oh thats cool. btw do you have any plans of adding kitsu api when adding episodes list to watch page? they provide a lot of episode details like episode image/description.

**Pull request merged**